### ISSUE TITLE
"postinstall": "ln -nsf ../src/js node_modules/app" fails when install from npm

### ISSUE THREAD

**Issue by PixelsCommander** [patriciogonzalezvivo/glslEditor#13]
"postinstall": "ln -nsf ../src/js node_modules/app" fails when install from npm . What is the purpose of this? Can it just be removed?

**Comment by brucelane** [comment]
it fails for me on Windows, ok on MacOs.
I don't know the purpose either...

**Comment by marcoose777** [comment]
Epic fail here too, unfortunately not experienced enough with npm/node.js to help debug

FYI: OS x86_64 Linux Arch (Manjaro)

Seems like the post-install script/line in the json file tries to create a symlink to a non-existent file called app. The warning lines don't look very optimistic either. Regressing to Kodelife.
  
output:
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue

> glslEditor@0.0.16 postinstall /home/spongebob/node_modules/glslEditor
> ln -nsf ../src/js node_modules/app

ln: failed to create symbolic link 'node_modules/app': No such file or directory
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN enoent ENOENT: no such file or directory, open '/home/spongebob/package.json'
npm WARN spongebob No description
npm WARN spongebob No repository field.
npm WARN spongebob No README data
npm WARN spongebob No license field.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! glslEditor@0.0.16 postinstall: `ln -nsf ../src/js node_modules/app`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the glslEditor@0.0.16 postinstall script 'ln -nsf ../src/js node_modules/app'.

**Comment by lagorsse** [comment]
Yep same issue here, I tried to remove it from local repo and install it with local dependencies, it's installed, but I have some Failed to load resource: net::ERR_FILE_NOT_FOUND

I think ln -nsf ../src/js node_modules/app should be ln -nsf ../src/js ../node_modules/app cause they are on the same level.

**Comment by hahaxiaowai** [comment]
so.. this issue be solved? How?
